test/functional/rpc_preciousblock.py

```diff
@@ -6,7 +6,7 @@
 
 from io import BytesIO
 
-from test_framework.test_framework import UnitETestFramework
+from test_framework.test_framework import UnitETestFramework, DISABLE_FINALIZATION
```

Comment:
Let's add a comment in the test, why we disable finalization here. I wonder why this PR requires to do so.

Reply:
Added a comment. It's because we create finalization and try to reorg before it. Before it took another epoch to reach finalization.